dotcom-rendering/Containerfile

```diff
@@ -4,11 +4,11 @@ FROM node:18.17.0-alpine
 
 WORKDIR /opt/app/dotcom-rendering/dotcom-rendering
 
-COPY ./dotcom-rendering/dist /opt/app/dotcom-rendering/dotcom-rendering/dist
+COPY ./dotcom-rendering/target /opt/app/dotcom-rendering/dotcom-rendering
```

Comment:
The removal of the zipping / taring of the bundle and then re-taring for riff-raff is a little unclear to me. I might not be following the reasoning but I wonder if we could keep the archive compressed and use the docker command ADD instead - which aiui will uncompress recognised archive formats:

https://docs.docker.com/engine/reference/builder/#add

Reply:
Container images have built in compression, so theres no need to compress before adding files to a container, that would just add complexity when running the container as you'd need to uncompress the files at runtime.

If we do the compression at the start of our build and then use `ADD` we'd be compressing the files, then immediately uncompressing them with ADD(it just uncompresses under the hood), and then recompress them when we publish to riffraff.

Reply:
Oh I had thought we might be able to avoid the compression step in publish.yml by keeping the artifact compressed and only uncompressing for runtime. Thanks for the info!